packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -26,8 +28,24 @@ export interface MenuProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'r
   'aria-label'?: string;
   /** Indicates if menu contains a flyout menu */
   containsFlyout?: boolean;
+  /** Indicates if menu contains a drilldown menu */
+  containsDrilldown?: boolean;
+  /** Indicates if a menu is drilled into */
+  isMenuDrilledIn?: boolean;
+  /** Indicates the drilldown path of menu itemIds */
+  drilldownPath?: string[];
```

Comment:
Maybe `drilldownPaths`?  The current name implies one path to me but the type is an array.

Reply:
My interpretation is that it is a path of nodes. Maybe `drilldownItemPath`?